src/client.js

```diff
@@ -1,25 +1,9 @@
-let isNode = false;
-
 const VERSION = '0.0.3';
 const RETRY_STATUS_CODES = [429, 500, 502, 503, 504];
 const ENDPOINT = 'https://api.mistral.ai';
 
-/**
- * Initialize fetch
- * @return {Promise<void>}
- */
-async function initializeFetch() {
-  if (typeof window === 'undefined' ||
-    typeof globalThis.fetch === 'undefined') {
-    const nodeFetch = await import('node-fetch');
-    fetch = nodeFetch.default;
-    isNode = true;
-  } else {
-    fetch = globalThis.fetch;
-  }
-}
-
-initializeFetch();
+const configuredFetch = globalThis.fetch ??
```

Comment:
This should work now in service workers (mv3 web ext), and in newer versions of node with fetch available natively